serial_protocol.md

```diff
@@ -0,0 +1,78 @@
+#Protocol:
+During the first use, or when the chiplet changes environments a “NETWORK” call is expected to initialize the wifi parameters.
+
+Every time a serial connection is established we will expect a “INIT” call after which any subsequent calls can be made, until the connection is closed. 
+
+In the following examples we use $ to represent variables. For example $Host represents your firebase host.
+
+##Response Format Byte
+All responses will be prefixed with one of the following bytes signifying the response type.
+```
+  + If response is ok and a raw string value.
+  * If response is ok and a raw string value prefixed by count of bytes in response then new line.
```

Comment:
> Ah, I see so you are more asking if we can merge this with "+"

Not really, all I'm saying is that when we return large size prefixed string, we might as well return them as "quoted" json string, therefor making the protocol simpler: small value are prefixed `+`, large value are size prefixed, json encoded and prefixed with '$'. Or did you want to be able to return large size prefixed raw string without json escaping?